### PR TITLE
Add SpaceLoot

### PR DESCRIPTION
**Pull request by horngep**
Added SpaceLoot: Starship-themed Loot on Terra

## Project Name

Terra Finder link: https://finder.terra.money/columbus-4/address/terra1ec833xj9f8dm0c04cdka9fkyefzgll7pdzzyxq

Website: https://spaceloot.xyz/

Twitter: https://twitter.com/spaceloot_nft

### CODE REVIEW

File: utils/lists.ts [hunk 1]
```diff
@@ -467,6 +467,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "sLoots are on-chain harlots and strumpets, for Adventurers' Pleasure",
     url: "https://sexyloot.co/",
   },
+  {
+    name: "SpaceLoot",
+    description: "Starship-themed Loot on Terra",
+    url: "https://spaceloot.xyz/",
```
Comment (zhew2013) [inline utils/lists.ts:473]: Your site seems down.

**Review by zhew2013 — CHANGES_REQUESTED** [review]
Just a comment on site availability.

**Comment by ravindranrahul** [comment]
hey @zhew2013 site seems up now!

**Comment by zhew2013** [comment]
Yup yup. @ravindranrahul thanks for rechecking! I will merge now.

**Comment by horngep** [comment]
thanks for merging! was wondering whats the usual schedule for deploying ? thanks a bunch!

**Pull request merged**